.eslintrc.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
     "eslint:recommended",
     "plugin:vue/recommended",
     "plugin:json/recommended",
-    "prettier", // Make sure "prettier" is the last element in this list.
+    // "prettier", // Make sure "prettier" is the last element in this list.
```

Comment:
Can we please not turn this off

Reply:
Sorry I didn't mean to commit this. I had to turn it off or else it wouldn't let me push due to an error in the screenshot I sent.